src/components/SearchComponent.tsx

```diff
@@ -20,9 +20,6 @@ const SearchComponent: FC<SearchComponentProps> = ({ onSearchChange, placeholder
   const navigate = useNavigate();
 
   const onSearch = useCallback(() => {
-    if (!searchString) {
```

Comment:
here I was wrong in the previous feature. Under this condition the search is not reset when an empty value is entered